Formula/mgis.rb

```diff
@@ -0,0 +1,37 @@
+class Mgis < Formula
+  desc "Provide tools to handle MFront generic interface behaviours"
+  homepage "https://thelfer.github.io/mgis/web/index.html"
+  url "https://github.com/thelfer/MFrontGenericInterfaceSupport/archive/refs/tags/MFrontGenericInterfaceSupport-2.0.tar.gz"
+  sha256 "cb427d77f2c79423e969815b948a8b44da33a4370d1760e8c1e22a569f3585e2"
+  license "GPL-1.0-or-later"
+  head "https://github.com/thelfer/MFrontGenericInterfaceSupport.git", using: :git, branch: "master"
+
+  depends_on "boost-python3" => :build
+  depends_on "cmake" => :build
+  depends_on "gcc" => :build
+  depends_on "python@3.9" => :build
+
+  def install
+    args = [
+      "-Denable-portable-build=ON",
+      "-Denable-website=OFF",
+      "-Denable-enable-doxygen-doc=OFF",
+      "-Denable-c-bindings=ON",
+      "-Denable-fortran-bindings=ON",
+      "-Denable-python-bindings=ON",  # requires boost-python
+      "-Denable-fenics-bindings=OFF", # experimental and very limited
+      "-Denable-julia-bindings=OFF",  # requires CxxWrap library
+      "-Denable-enable-static=OFF",
+    ]
```

Comment:
Why can't we use default settings?

Reply:
The default settings generate documentation, a web site and do not compile python or c bindings for example. I think these settings are most relevant to most users.